containers/search/Searchbox.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { c } from 'ttag';
+import PropTypes from 'prop-types';
+import { SearchInput, Icon } from 'react-components';
+
+import { classnames } from '../../helpers/component';
+
+const Searchbox = ({ className = '', placeholder = '', value = '', onSearch }) => {
+    return (
+        <div className={classnames(['searchbox-container relative flex-item-centered-vert', className])}>
+            <label htmlFor="global_search">
+                <span className="sr-only">{placeholder}</span>
+                <SearchInput
+                    value={value}
+                    onChange={onSearch}
+                    id="global_search"
+                    placeholder={placeholder}
+                    className="searchbox-field"
+                />
+            </label>
+            <button type="button" className="searchbox-search-button flex">
+                <Icon name="search" className="fill-white mauto searchbox-search-button-icon" />
+                <span className="sr-only">{c('Action').t`Search`}</span>
+            </button>
```

Comment:
You should add also support for an `advanced` button.
![Capture d’écran 2019-10-07 à 10 49 25](https://user-images.githubusercontent.com/1345786/66297527-2611b300-e8f0-11e9-922e-336953dd17a3.png)